types/setting.go

```diff
@@ -41,6 +40,9 @@ const (
 	SettingTypeInt        = SettingType("int")
 	SettingTypeBool       = SettingType("bool")
 	SettingTypeDeprecated = SettingType("deprecated")
+
+	ValueIntRangeMinimumKey = "minimum"
+	ValueIntRangeMaximumKey = "maximum"
```

Comment:
Why not just `ValueIntRangeMinima` and `ValueIntRangeMaxima`?

Reply:
Do we need to handle `close bracket` and `open bracket` such as `(min, max)`, `[min, max]`, `(min, max]` and `[min, max)`

Reply:
I think maximum, minimum is the correct spelling
Yeah I can remove the "key" suffix, maybe more easy to use when coding.

Reply:
I don't think we need to handle `close bracket` and `open bracket`
That is too complicated.
Besides, when we say `maximum value`, `minimum` value
We usually include the value, and we do it in our original code.

For example:
- Our original Code:
```
	case SettingNameStorageMinimalAvailablePercentage:
		if _, err := strconv.Atoi(value); err != nil {
			return errors.Wrapf(err, "value %v is not a number", value)
		}
		// additional check whether minimal available percentage is between 0 to 100
		value, err := util.ConvertSize(value)
		if err != nil {
			return errors.Wrapf(err, "failed to parse %v as size", value)
		}
		if value < 0 || value > 100 {
			return fmt.Errorf("value %v should between 0 to 100", value)
		}
```
The minimum value, maximum value are included: [0, 100]
We check if the `value < minimum || value > max`
I think this convention is common to user and developer

Reply:
> I think maximum, minimum is the correct spelling Yeah I can remove the "key" suffix, maybe more easy to use when coding.

Yeah, I mean `key` seems not needed.

Reply:
> I don't think we need to handle `close bracket` and `open bracket` That is too complicated. Besides, when we say `maximum value`, `minimum` value We usually include the value, and we do it in our original code.

OK. Just make sure if the `close bracket` is required. If not, I'm fine with the implementation.

Reply:
Fixed, thanks